app/src/main/java/camp/nextstep/edu/tictactoe/MainActivity.kt

```diff
@@ -4,16 +4,38 @@ import android.os.Bundle
 import android.view.Menu
 import android.view.MenuItem
 import android.widget.Toast
+import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
 import camp.nextstep.edu.tictactoe.databinding.ActivityMainBinding
+import com.nextstep.edu.tictactoe.domain.Winner
 
 class MainActivity : AppCompatActivity() {
+    private val viewModel: TictactoeViewModel by viewModels()
     private lateinit var binding: ActivityMainBinding
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
+        binding.lifecycleOwner = this
+        binding.tictactoeViewModel = viewModel
```

Comment:
```suggestion
        with (binding) {
	    lifecycleOwner = this
	    binding.tictactoeViewModel = viewModel
        }
```
이미 아실 수도 있겠지만 with 블록을 이용하면 이러한 값들을 묶어둘 수 있습니다! 꼭 반영하진 않아도 됩니다 🙂

Reply:
음... 이렇게 하면 this가 ActivityMainBinding이 되어서 lifecycleOwner 할당이 안되네요. 가능한 방법이 있는지 찾아봤지만 잘 보이지 않아서 현 상태 유지하겠습니다. 🙂